hpOneView/oneview_client.py

```diff
@@ -116,7 +116,8 @@ class OneViewClient(object):
     DEFAULT_API_VERSION = 300
 
     def __init__(self, config):
-        self.__connection = connection(config["ip"], config.get('api_version', self.DEFAULT_API_VERSION), config.get('ssl_certificate', False))
+        self.__connection = connection(config["ip"], config.get('api_version', self.DEFAULT_API_VERSION),
+                                       config.get('ssl_certificate', False), config.get('timeout'))
```

Comment:
Is there a reason to move `config.get('ssl_certificate', False),`? Otherwise it should remain in [line 119](https://github.com/HewlettPackard/python-hpOneView/pull/341/files#diff-ac2fcb47ec5c67f3daeef6ea682c449eR119).

Reply:
Done